### ISSUE TITLE
Dynamic Symbol Retrieval Error: libtdjson.so: undefined symbol: td_create_client

### ISSUE THREAD

**Issue by uWynell** [airgram/airgram#138]
**Issue type:**

[x] question
[ ] bug report
[ ] feature request
[ ] documentation issue

**Airgram version:**

[x] `latest`
[ ] `@next`
[ ] `2.3.x` (or put your version here)

**Steps to reproduce or a small repository showing the problem:**

```ts
import { Airgram, Auth, prompt } from 'airgram';
import { resolve } from 'path';

const client = new Airgram({
  apiHash: '', // api hash here
  apiId: 0, // api id here
  command: '/home/wynell/td/tdlib/lib/libtdjson' // file: /home/wynell/td/tdlib/lib/libtdjson.so
});

client.use(new Auth({
  phoneNumber: () => prompt('Enter your phone number: '),
  code: () => prompt('Enter the code you received: '),
  password: () => prompt('Enter your password: ')
}));

client.on('updateNewMessage', (m) => {
  console.log({ message: m });
});
```

Then I run: `ts-node main.ts` and get the following output:
```
/home/wynell/irbot/node_modules/ffi-napi/lib/dynamic_library.js:112
    throw new Error('Dynamic Symbol Retrieval Error: ' + this.error());
          ^
Error: Dynamic Symbol Retrieval Error: /home/wynell/td/tdlib/lib/libtdjson.so: undefined symbol: td_create_client
    at DynamicLibrary.get (/home/wynell/irbot/node_modules/ffi-napi/lib/dynamic_library.js:112:11)
    at /home/wynell/irbot/node_modules/ffi-napi/lib/library.js:50:21
    at Array.forEach (<anonymous>)
    at Object.Library (/home/wynell/irbot/node_modules/ffi-napi/lib/library.js:47:28)
    at new TdJsonClient (/home/wynell/irbot/node_modules/airgram/components/TdJsonClient.js:86:27)
    at new Airgram (/home/wynell/irbot/node_modules/airgram/Airgram.js:50:116)
    at Object.<anonymous> (/home/wynell/irbot/src/index.ts:7:17)
    at Module._compile (node:internal/modules/cjs/loader:1102:14)
    at Module.m._compile (/mnt/c/Program Files/nodejs/node_modules/ts-node/src/index.ts:1043:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1131:10)
```

Yes, I also tried to compile the source code using `tsc` typescript compiler, and got the same output.

How to fix that?

**Comment by esindger** [comment]
Probably, you are using old TDLib build. Airgram `v 3.2.*` is required TDLib at least `v 1.6.9`. You may find more information [here](https://github.com/airgram/airgram/issues/130).

**Comment by uWynell** [comment]
> Probably, you are using old TDLib build. Airgram `v 3.2.*` is required TDLib at least `v 1.6.9`. You may find more information [here](https://github.com/airgram/airgram/issues/130).

I rebuild it a few hours ago.

**Comment by esindger** [comment]
I experienced the same problem (build for macOS) a month ago and created [an issue](https://github.com/tdlib/td/issues/1261). That bug was immediately fixed.

If you are sure that you use the latest version of TDLib, try to create a new issue in [TDLib repository](https://github.com/tdlib/td/issues).

**Comment by jumoog** [comment]
newest TDLib changed in commit https://github.com/tdlib/td/commit/5eea5b75010fe6649254faba1ca7a879f93eb8cc
`td_create_client` -> `td_create_client_id`

you have to change in `td_create_client` -> `td_create_client_id` node_modules/airgram/components/TdJsonClient.js

so change:
`return this.client.td_create_client();`
`td_create_client: ['int', []],`

to:
`return this.client.td_create_client_id();`
`td_create_client_id: ['int', []],`

**Comment by esindger** [comment]
Fixed in `v3.3.0`
